Tests/scripts/jira_issues.py

```diff
@@ -62,45 +62,51 @@ def jira_color_text(text: str, color: str) -> str:
     return f"{{color:{color}}}{text}{{color}}"
 
 
+def find_jira_ticket_unresolved_transition(jira_server: JIRA, jira_issue: Issue | None):
+    unresolved_transition = None
+    #  Get the available transitions for the issue
+    transitions = jira_server.transitions(jira_issue)
+    for transition in transitions:
+        if transition['name'] == JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME:
+            unresolved_transition = transition
+            break
+    return unresolved_transition['id'] if unresolved_transition else None
+
+
+def transition_jira_ticket_to_unresolved(jira_server: JIRA, jira_issue: Issue | None, unresolved_transition_id: str | None):
+    if unresolved_transition_id:
+        jira_server.transition_issue(jira_issue, unresolved_transition_id)
+
+
```

Comment:
why did you split into 2 functions? honestly i don't see the point.

Reply:
This allows me to make only one call to the jira_server.transitions function